### PR TITLE
[13.0][MIG] crm_claim

### PR DESCRIPTION
**Pull request by ludolelarge**
[13.0][MIG] crm_claim

### CODE REVIEW

**Comment by ludolelarge** [comment]
I have some errors in travis ... isort ... black ... Is there someone can getting me some help ?

**Comment by rousseldenis** [comment]
@ludolelarge Please follow : https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

**Comment by rousseldenis** [comment]
@ludolelarge Please read carefully the link I sent you. You need to preserve commit history.

**Comment by rousseldenis** [comment]
Don't lose time formatting each file individually. Use pre-commit to reformat all files automatically as described in the link I sent.

**Comment by ludolelarge** [comment]
> 
> 
> @ludolelarge Please follow : https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

Thx for help ... Sorry for the question but im noob
Finaly i use command line to migrate but i have this error

ludovic@Odoo:~/Github$ git checkout -b 13.0-mig-crm_claim_claim origin/13.0
fatal: Not a git repository (or any of the parent directories): .git

**Comment by rousseldenis** [comment]
> > @ludolelarge Please follow : https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0
> 
> Thx for help ... Sorry for the question but im noob
> Finaly i use command line to migrate but i have this error
> 
> ludovic@Odoo:~/Github$ git checkout -b 13.0-mig-crm_claim_claim origin/13.0
> fatal: Not a git repository (or any of the parent directories): .git

Ok, no problem. You would maybe look for information about git with your prefered search engine. e.g.: https://product.hubspot.com/blog/git-and-github-tutorial-for-beginners

The problem you have is that you haven't cloned the repository to your local dir.

File: crm_claim/README.rst [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fcrm-lightgray.png?logo=github
+    :target: https://github.com/OCA/crm/tree/12.0/crm_claim
```
Comment (manuelcalerosolis) [inline crm_claim/README.rst:17]: change to 13.0 version

File: crm_claim/models/crm_claim.py [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+        "Separate multiple email addresses with a comma",
+    )
+    email_from = fields.Char(string="Email", help="Destination email for email gateway.")
+    partner_phone = fields.Char(string="Phone")
+    stage_id = fields.Many2one(
+        comodel_name="crm.claim.stage",
+        string="Stage",
+        track_visibility="onchange",
+        default=_get_default_stage_id,
+        domain="['|', ("team_ids", "=", team_id), ("case_default", "=", True)]"
```
Comment (manuelcalerosolis) [inline crm_claim/models/crm_claim.py:99]: change this line to :

"['|', ('team_ids', '=', team_id), ('case_default', '=', True)]"

File: crm_claim/models/res_partner.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+    @api.depends("claim_ids", "child_ids", "child_ids.claim_ids")
+    def _compute_claim_count(self):
+        partners = self | self.mapped("child_ids")
+        partner_data = self.env["crm.claim"].read_group(
+            [("partner_id", "in", partners.ids)], ["partner_id"], ["partner_id"]
+        )
+        mapped_data = {
+            m["partner_id"][0]: m["partner_id_count"] for m in partner_data
+        }
```
Comment (manuelcalerosolis) [inline crm_claim/models/res_partner.py:22]: only in ole line :

mapped_data = {m["partner_id"][0]: m["partner_id_count"] for m in partner_data}

Comment (pedrobaeza) [reply crm_claim/models/res_partner.py:22]: Don't worry about these things, as they are automatically done by black.

**Comment by manuelcalerosolis** [comment]
@ludolelarge we are very interested in the migration of this module, if you want we can continue with the migration, based on your changes.

thanks

**Comment by flachica** [comment]
It's possible that I need this module in v13. Is there any news on the state of migration? 

I also think it would be appropriate to create instances of mail.activity instead of sending direct emails. For example assigning a mail.activity to crm.claim.category so that it takes the team leader as the receiver of alerts. The objective would be that, in addition to sending the email, Odoo would have a notice when the task assigned to the claim expires to improve its monitoring.

**Pull request closed without merging**